backend/app/views/spree/admin/shared/_locale_selection.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="admin-locale-selection">
```

Comment:
Maybe we should put this div into the `if available_locales.size > 1`? I think there's no reason to keep the html if empty.

Reply:
done.